geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -169,17 +167,20 @@ abstract class AbstractGeoIngestProcessor(dataStoreProperties: Seq[PropertyDescr
   }
 
   @OnRemoved
-  @OnDisabled
+  @OnStopped
+  @OnShutdown
   def cleanup(): Unit = {
-    if (converterPool != null) {
-      CloseWithLogging(converterPool)
-      converterPool = null
```

Comment:
Is there any converter pooling still going on?

Reply:
yeah, it's just down on line 419: https://github.com/geomesa/geomesa-nifi/pull/53/files#diff-460fdeb97359e630d090e2f325a18dc5R419